### PR TITLE
ref(grouping): Remove CalculatedHashes

### PR DESCRIPTION
**Pull request by armenzg**
Code originally added in https://github.com/getsentry/sentry/pull/26740

### CODE REVIEW

File: src/sentry/grouping/ingest/hashing.py [hunk 1]
```diff
@@ -85,7 +83,6 @@ def _calculate_event_grouping(
                 event.data["grouping_config"] = get_grouping_config_dict_for_project(project)
                 hashes = event.get_hashes()
 
-        hashes.write_to_event(event.data)
```
Comment (armenzg) [inline src/sentry/grouping/ingest/hashing.py:88]: `get_hashes()` now takes care of writing to the event:
image

File: src/sentry/grouping/ingest/hashing.py [hunk 2]
```diff
@@ -239,22 +235,17 @@ def get_hash_values(
         secondary_hashes,
     )
 
-    all_hashes = CalculatedHashes(
-        hashes=extract_hashes(primary_hashes) + extract_hashes(secondary_hashes),
-        # We don't set a combo `variants` value here because one set of variants would/could
-        # partially or fully overwrite the other (it's a dictionary), and having variants from two
-        # different configs all mixed in together makes no sense.
+    return (
```
Comment (JoshFerge) [inline src/sentry/grouping/ingest/hashing.py:238]: is it necessary to return the third combined item here? can the callers just concatenate them?

**Review by JoshFerge — APPROVED** [review]
makes sense to me. might be good for @lobsterkatie to take a look as well.

File: src/sentry/grouping/ingest/hashing.py (hunk 2 above)
Comment (armenzg) [reply src/sentry/grouping/ingest/hashing.py:238]: Good point. I will look into it.

**Comment by armenzg** [comment]
> makes sense to me. might be good for @lobsterkatie to take a look as well.

@JoshFerge ugh. Want me to revert? Or wait to hear back?

**Comment by JoshFerge** [comment]
@armenzg i think you're fine.

**Pull request merged**